Document/0x04c-Tampering-and-Reverse-Engineering.md

```diff
@@ -95,3 +95,30 @@ In the late 2000s, testing based on symbolic execution has become a popular way
 Typically, symbolic execution is combined with other techniques such as dynamic execution to mitigate the path explosion problem specific to classical symbolic execution. This combination of concrete (actual) and symbolic execution is referred to as *concolic execution* (the name concolic stems from *conc*rete and symb*olic*. Together with improved SMT solvers and current hardware speeds, concolic execution allows to explore paths in medium-size software modules (i.e., on the order of 10s KLOC). However, it also comes in handy for supporting de-obfuscation tasks, such as simplifying control flow graphs. For example, Jonathan Salwan and Romain Thomas have [shown how to reverse engineer VM-based software protections using Dynamic Symbolic Execution](https://triton.quarkslab.com/files/csaw2016-sos-rthomas-jsalwan.pdf "Jonathan Salwan and Romain Thomas: How Triton can help to reverse virtual machine based software protections") (i.e., using a mix of actual execution traces, simulation, and symbolic execution).
 
 In the Android section, you'll find a walkthrough for cracking a simple license check in an Android application using symbolic execution.
+
+### References
+
+#### OWASP Mobile Top 10 2016
+
+- [M9 - Reverse Engineering](https://www.owasp.org/index.php/Mobile_Top_10_2016-M9-Reverse_Engineering "M9 - Reverse Engineering")
+
+#### OWASP MASVS
+- V8.1: "The app detects, and responds to, the presence of a rooted or jailbroken device either by alerting the user or terminating the app."
```

Comment:
are we sure this belongs here?

Reply:
Good point. As we do not describe test cases in detail that match to MASVS in this chapter, it doesn't make much sense to reference to the MASVS. Removed it, but kept the reference to Mobile top 10 and the tool listing that are referenced in this section. Just wanted to be consistent with the other sections in the general testing guide that also have a reference section